R/reportCrossSubmodels.R

```diff
@@ -0,0 +1,148 @@
+#' Read a finished report and calculate variables that need not only variables produced by other
+#' report*.R functions, (from convGDX2MIF.R), but also variables coming from submodels for
+#' which the reporting is done outside of it, such as EDGE-T and MAGICC.
+#' #'
+#'
+#' @param inmifpath a character string with the path to a REMIND mif file, that
+#' should contain only one model and scenario, and already have both the REMIND and EDGE-T variables
+#'
+#' @author Gabriel Abrahao
+#' @export
+#' @importFrom assertr assert not_na
+#' @importFrom gdx readGDX
+#' @importFrom magclass getYears getRegions mbind setNames mselect
+#' new.magpie setYears mcalc dimReduce
+#' @importFrom tibble as_tibble
+#' @importFrom tidyselect everything
+#' @importFrom tidyr everything
+#' @importFrom madrat readSource toolGetMapping toolAggregate
+#' @importFrom mrcommons everything
+#' @importFrom piamutils deletePlus
+#'
+
+reportCrossSubmodels <- function(inmifpath) {
```

Comment:
In which context do you want to execute this function? If it is part of convGDX2MIF, it should rather use the gdx as input. If it should run outside of convGDX2MIF (which I assume is the case), it could be a function here or a script in REMIND. I am not sure what is better, need more context for the execution of this function.